### ISSUE TITLE
Add PyTorch-UNet model to benchmark repo

### ISSUE THREAD

**Issue by wconstab** [pytorch/benchmark#91]
We're building a suite of pytorch benchmarks by forking popular open-source models and modifying them to conform to a common API that facilitates a central tool installing, running, and collecting measurements.

Benchmark Repo: https://github.com/pytorch/benchmark
Instructions: https://github.com/pytorch/benchmark/blob/master/models/ADDING_MODELS.md
Source model: https://github.com/supriyar/Pytorch-UNet

The UNet model has already been forked and modified in a previous phase of this process, but the API (install.py, hubconf.py) had not yet been standardized.  Copy the repo into benchmark.git, follow the latest instructions in the ADDING_MODELS readme to add the new API. Note, the 'install.sh' and 'run.sh' scripts added in the previous effort would serve as good hints on how to proceed.

**Comment by Krovatkin** [comment]
@wconstab I'll try to do this one!

**Comment by msaroufim** [comment]
Hi @Krovatkin @wconstab are you still interested in doing this? I was looking for a bootcamp task to work on tomorrow and I've already worked with this specific Unet implementation in the past

**Comment by Krovatkin** [comment]
@msaroufim, yes, sounds like a great bootcamp task. Unfortunately, I didn't get very far :(

**Comment by msaroufim** [comment]
Sweet! I’ll take a look
